docs/utils/quick-start.md

```diff
@@ -10,7 +10,7 @@ description: Getting started with the Utils package in React Native Firebase
 Install this module with Yarn:
 
 ```bash
-yarn add @react-native-firebase/utils
+yarn add @react-native-firebase/app
```

Comment:
I guess we can remove this directory now, or at least migrate any docs into the app directory?